tests/test_responsetypes.py

```diff
@@ -87,33 +91,39 @@ def test_from_args_pre_xtractmime(self):
             ({'url': 'http://www.example.com/item/file.html'}, HtmlResponse),
             ({'body': b'<html><head><title>Hello</title></head>'}, HtmlResponse),
             ({'body': b'<?xml version="1.0" encoding="utf-8"'}, XmlResponse),
-            ({'filename': 'file.pdf'}, Response),
-            ({'url': 'http://www.example.com/item/file.pdf'}, Response),
+            ({'filename': 'file.pdf'}, TextResponse),
+            ({'url': 'http://www.example.com/item/file.pdf'}, TextResponse),
```

Comment:
I see some scenarios here that seem to belong to post_xtractmine. Please, review the scenarios that belong to each function, as the release notes will describe changes based on these 2 test functions.